aip/general/0149.md

```diff
@@ -0,0 +1,41 @@
+---
+id: 149
+state: reviewing
+created: 2021-02-12
+placement:
+  category: fields
+  order: 95
+---
+
+# Unset field values
+
+In most resources and structs, many fields are optional: the user is not
```

Comment:
Does "struct" here have a specific meaning? Are you talking about google.protobuf.Struct? (I'm assuming not.)

Perhaps just "In most messages"?

Reply:
Done.